seatunnel-e2e/seatunnel-connector-v2-e2e/connector-paimon-e2e/src/test/java/org/apache/seatunnel/e2e/connector/paimon/PaimonSinkCDCIT.java

```diff
@@ -636,7 +636,7 @@ public void testChangelogLookup(TestContainer container) throws Exception {
                         .collect(Collectors.toList());
         log.info("paimon records: {}", actual2);
         Assertions.assertEquals(2, actual2.size());
-        Assertions.assertEquals(Arrays.asList("[+U, 1, Aa, 200]", "[+I, 2, Bb, 90]"), actual2);
+        Assertions.assertEquals(Arrays.asList("[+I, 1, Aa, 200]", "[+I, 2, Bb, 90]"), actual2);
```

Comment:
I think your test case may he not all right. Your test case which paimon to paimon in streaming mode only read the data files. Currently, We have not supported read the changelog file in paimon source reader. We just only support streaming read. If you want to read changelog file, you should use it like this:

<img width="1260" alt="image" src="https://github.com/user-attachments/assets/08e79071-6289-4a46-a230-996740e7e720">